### ISSUE TITLE
Configuration for rule "jsx-a11y/label-has-for" is invalid: Value "label" should be object. 

### ISSUE THREAD

**Issue by realtebo** [reactjs/react-a11y#141]
```
c:\Users\realtebo\Downloads\manager\node_modules\eslint-config-rallycoding\rules\react-a11y.js: Configuration for rule "jsx-a11y/label-has-for" is invalid: Value "label" should be object. 

Referenced from: c:\Users\realtebo\Downloads\manager\node_modules\eslint-config-rallycoding\index.js 
Referenced from: c:\Users\realtebo\Downloads\manager\.eslintrc
```

**Comment by erin-doyle** [comment]
Can you provide any more detail here?  What is the source code that you're getting this warning on?  Do you think it's wrong?

**Comment by erin-doyle** [comment]
I'm going to close this since there's not enough information provided to go on.  Please reopen with an example of the source code causing this error if you're still running into this.
